functions/replace-route/app.py

```diff
@@ -358,7 +358,7 @@ def connectivity_test_handler(event, context):
     logger.info("Starting NAT instance connectivity test")
 
     check_interval = int(os.getenv("CONNECTIVITY_CHECK_INTERVAL", DEFAULT_CONNECTIVITY_CHECK_INTERVAL))
-    check_urls = os.getenv("CHECK_URLS", DEFAULT_CHECK_URLS)
+    check_urls = os.getenv("CHECK_URLS", DEFAULT_CHECK_URLS).split(",")
```

Comment:
You need to update `DEFAULT_CHECK_URLS` also

Reply:
Monday after Sunday of re:Invent is always rough.